src/uiHelpers.c

```diff
@@ -125,8 +125,8 @@ void displayScrollingText(
 	os_memset(&displayState, 0, sizeof(displayState));
 
 	// Copy data
-	os_memmove(scrollingState->header, header, header_len);
-	os_memmove(scrollingState->fullText, text, text_len);
+	os_memmove(scrollingState->header, headerStr, header_len);
```

Comment:
Please, double check that we have space for terminating zero and copy +1 byte ... (I know we are memsetting at the beginning but let's make the behavior expected)